### ISSUE TITLE
Update zh-hans translation for 0.2.0

### ISSUE THREAD

**Issue by birtles** [birtles/rikaichamp#363]
The set of changes here:

https://github.com/birtles/rikaichamp/compare/bd9887525a..5dcdff7

Roughly it is:

* Add `options_dictionary_data_heading`
* Rename `options_kanji_data_source_no_version` to `options_data_source`, add reference to `JMnedict/ENAMDICT` and drop the version number and generated time.
* Drop `options_kanji_data_source_with_version`
* Rename `options_kanji_license` to `options_edrdg_license` (and update the description to refer to `options_edrdg_license_keyword`)
* Rename `options_kanji_license` to `options_edrdg_license_keyword`
* Add `options_kanji_data_title`
* Add `options_name_data_title`
* Add `options_data_series_version_and_date`
* Add `options_data_series_date_only`
* Drop `options_kanji_db_version`
* Add `options_name_data_name`
* Update `options_updating_data` to include progress information

@SaltfishAmi can you help with this?

**Comment by birtles** [comment]
Oh, also, I renamed the `master` branch to `main` so you might need to update any local checkout you have as per: https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx, i.e.

```bash
$ git checkout master
$ git branch -m master main
$ git fetch
$ git branch --unset-upstream
$ git branch -u origin/main
$ git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main
```

Or just re-clone the repo.

**Comment by melink14** [comment]
Oh, nice on already renaming master! I'll have to check out that link since
I was going to do it but Github had a warning to wait until they made it
automatic or something.

**Comment by SaltfishAmi** [comment]
I'll look into this!

**Comment by birtles** [comment]
> Sure, I'll look into this later, maybe in a few days.

Thanks! I might go ahead and release v0.2.0 and then do a dot release when the zh-hans localization is ready (although I should probably drop the keys where the number of arguments has changed to avoid runtime errors--or just replace them with the English text for now).

**Comment by birtles** [comment]
Ok, I lined up the keys and just filled in the English text for now. All that remains is:

* `options_dictionary_data_heading` needs translation
* `options_data_source` needs to mention JMnedict/ENAMDICT
* `options_kanji_data_title` needs translation
* `options_name_data_title` needs translation
* `options_data_series_version_and_date` needs translation
* `options_data_series_date_only` needs translation
* `options_name_data_name` needs translation

**Comment by birtles** [comment]
Closed by #370
